libs/partners/pinecone/tests/unit_tests/test_embeddings.py

```diff
@@ -0,0 +1,29 @@
+"""Test embedding model integration."""
+
+from langchain_core.embeddings import Embeddings
+
+from langchain_pinecone import PineconeEmbeddings
+
+MODEL = "multilingual-e5-large"
+MODEL2 = "multilingual-e5-small"
```

Comment:
I don't think we support the e5-small model yet, just fyi

Reply:
thanks – I'm aware, just added as an example for unit tests